cli/src/main/java/org/jboss/as/cli/operation/OperationRequestCompleter.java

```diff
@@ -487,6 +487,14 @@ protected int complete(CommandContext ctx, ParsedCommandLine parsedCmd, Operatio
                 candidates.set(0, Util.escapeString(candidate, ESCAPE_SELECTOR) + '=');
             }
         } else {
+            if (address.endsOnType()) {
```

Comment:
This code is induced by the var being in candidates when completing. You shouldn't need this change. This is the kind of change that, when we read the code, we don't understand why it is there. Difficult to maintain when a futur bug will be chased.